tests/ae-inventory-subjects-itest/src/test/java/org/metaeffekt/core/itest/common/AbstractPreparer.java

```diff
@@ -40,7 +40,7 @@ public Preparer setSource(String url) {
 
     @Override
     public Preparer setName(String testname) {
-        this.name = testname;
+        this.name = testname.replace("org.metaeffekt.core.itest.","");
```

Comment:
We need to anticipate the final organization of modules and file here. This will not always be the same package string here.